### PR TITLE
fix(Twitter): cache tokens

### PR DESCRIPTION
**Pull request by UncleBill**
Extracting tokens from script file is expensive.

### CODE REVIEW

File: packages/web3-providers/src/twitter/index.ts [hunk 1]
```diff
@@ -67,13 +80,15 @@ async function getTokens(operationName?: string) {
...
         : undefined
-
-    return {
+    tokens = {
         bearerToken,
         queryToken,
         csrfToken,
         queryId,
     }
+    tokenCache.set('tokens', tokens)
```
Comment (guanbinrui) [inline packages/web3-providers/src/twitter/index.ts:89]: Why choose LRU?

Comment (UncleBill) [reply packages/web3-providers/src/twitter/index.ts:89]: It's Last Recent Use. Any other recommendations

Comment (guanbinrui) [reply packages/web3-providers/src/twitter/index.ts:89]: It's overkill. Maybe an in-memory local variable is enough.

Comment (UncleBill) [reply packages/web3-providers/src/twitter/index.ts:89]: Well, LRU can control cache expiration.

Comment (Jack-Works) [reply packages/web3-providers/src/twitter/index.ts:89]: use LRU but you only store 1 item? that's strange

Comment (UncleBill) [reply packages/web3-providers/src/twitter/index.ts:89]: Well, it's better to isolate it. If you mix it into another cache object, when the number of items reached `max`, our cached tokens might be cleared.

**Review by Jack-Works — DISMISSED** [review]

**Pull request closed without merging**